Sources/SkipUI/SkipUI/View/View.swift

```diff
@@ -271,9 +271,15 @@ extension View {
         #endif
     }
 
-    @available(*, unavailable)
     public func colorInvert() -> some View {
+        #if SKIP
+        return ComposeModifierView(targetView: self) {
+            $0.modifier = $0.modifier.then(Modifier.colorInvert())
```

Comment:
This could just be $0.modifier.colorInvert(), correct? I might tweak this before tagging the release